src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,11 +5,43 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private double armLength;
+    private double bucketCapacity;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator` class. You should define whether these fields are private, protected, or public (though private is most common and recommended here) to encapsulate the data properly.